website/versioned_docs/version-2.0.0-alpha.63/theme-classic.md

```diff
@@ -304,6 +304,24 @@ module.exports = {
 };
 ```
 
+### Navbar force style
+
+This might be useful if you prefer static Navbar design without disabling the theme switching ability. You can force single theme to always apply no matter which theme user have selected.
+
+Currently, two possible style options are `dark` and `primary`.
```

Comment:
Wording like "this might" seems bad for doc to me. We should rather reference the Infima doc directly so that user can understand better (https://facebookincubator.github.io/infima/docs/components/navbar/#dark)

Maybe we could actually just put a "className" config attribute, and then user would just force any class he wants? Looks like a flexible and future proof option, so the user would just use `className: 'navbar--primary'` in config?